src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -306,18 +306,18 @@ export default class FilterOptionsComponent extends Component {
       // this.name otherwise, instead opt for this.core.setStaticFilterNodes()
```

Comment:
I belive this comment needs to be updated to say StorageKeys.HISTORY_POP_STATE rather than this.name

Reply:
ah good catch I'll just remove this comment